packages/components/ui/src/drivers/fonts/stores.js

```diff
@@ -0,0 +1,73 @@
+import * as _ from 'lamb';
+import {makeMergeAppliedFnMap} from '@svizzle/utils';
+import {derived, get, writable} from 'svelte/store';
+
+// fonts utils
+const ungroupFamily = (family, faces) => faces.map(face => new FontFace(
+	family,
+	face.src,
+	face.descriptors
+));
+const isFirstToLoad = firstFamilyToLoad => _.pipe([
+	_.getKey('family'),
+	_.is(firstFamilyToLoad)
+]);
+
+const getRuntimeFamily = _.adapter([
+	_.casus(_.hasKey('faces'), _.pipe([
+		_.collect([
+			_.getKey('family'),
+			_.getKey('faces')
+		]),
+		_.apply(ungroupFamily),
+	])),
+	_.always([])
+]);
+
+const getRuntimeFamilies = _.mapWith(getRuntimeFamily);
+
+// main stores
+export const _firstFamilyToLoad = writable('');
+export const _fontFamilies = writable({});
+export const _status = writable({
+	isFirstLoaded: false,
+	isDone: false
+})
+
+// deriveds
+export const _fontFamilyNames = derived(
+	_fontFamilies,
+	_.mapWith(_.getKey('family'))
```

Comment:
(1) 

I would have a new file `packages/components/ui/src/drivers/fonts/utils.js` with:

```
export const getFamily = _.getKey('family');
export const getFamilyNames = _.mapWith(getFamily);
```

and any utility we use here, so that we would import `getFamilyNames` from `@svizzle/ui/src/drivers/fonts/utils` in the app.